lib/seattleflu/id3c/cli/command/etl/clinical.py

```diff
@@ -304,6 +309,78 @@ def flu_shot(flu_shot_response: Optional[Any]) -> list:
 
     return [flu_shot_map[flu_shot_response]]
 
+def admit_encounter(admit_encounter_response: Optional[Any]) -> list:
+    """
+    Given a *admit_encounter_response*, returns yes/no value for AdmitDuringThisEncounter key.
+
+    >>> admit_encounter(0.0)
+    ['no']
+
+    >>> admit_encounter('TRUE')
+    ['yes']
+
+    >>> admit_encounter('maybe')
+    Traceback (most recent call last):
+        ...
+    id3c.cli.command.etl.UnknownAdmitEncounterResponseError: Unknown admit during this encounter response «maybe»
```

Comment:
This string needs to match exactly with that the function is returning below (
"Unknown admit during this encounter response " doesn't match "Unknown admit during encounter response ")